test/parser/test-run-test262.js

```diff
@@ -437,7 +437,8 @@ const failExcluded = [
   '682.script.js',
 ];
 
-const earlyExcluded = [ // eslint-disable-line no-unused-vars
+const earlyExcluded = [
+  // eslint-disable-line no-unused-vars
```

Comment:
Prettier is in conflict w/ eslint comment directives? There's an issue for it https://github.com/prettier/prettier/issues/1267

Should we hold off and see if there's a fix first? Or do you want to use the workaround suggested in the issue?